tests/ensemble_test.py

```diff
@@ -78,9 +87,13 @@ def test_c2st_posterior_ensemble_on_linearGaussian(inference_method, num_trials)
     )
     target_samples = gt_posterior.sample((num_samples,))
 
-    simulator, prior = prepare_for_sbi(
-        lambda theta: linear_gaussian(theta, likelihood_shift, likelihood_cov), prior
+    prior, _, prior_returns_numpy = process_prior(prior)
```

Comment:
same here, no need to use these helpers here. just use the prior and simulator as they are.